sentry-samples/kmp-app/iosApp/iosApp/ContentView.swift

```diff
@@ -4,6 +4,17 @@ import shared
 struct ContentView: View {
 	var body: some View {
 		Text("KPM Sample App " + Platform().platform)
+        VStack() {
+            Button("Capture Message") {
+                SentryKMP().captureMessage(message: "KMP Sample App " + Platform().platform)
```

Comment:
`SentryKMP` should be static.

Reply:
It is static on Kotlin but due to how KMM creates those Obj-C bindings it doesn't seem to be able to create a static callable class in Swift. I'll check if there's a workaround

Reply:
```Swift
SentryKMP() === SentryKMP()
```

seems to point to the same instance in Swift, so maybe it's fine to use it like this but document it ?